### ISSUE TITLE
parameter-list-wrapping expects wrong indentation in returning if-else condition

### ISSUE THREAD

**Issue by romtsn** [pinterest/ktlint#864]
## Expected Behavior
Expect this code to not cause any errors
```Kotlin
fun function() {
    return if (listOfNotNull(
            param1,
            param2,
            param3
        ).isEmpty()
    ) {
        true
    } else {
        false
    }
}
```

## Observed Behavior
```Kotlin
...
Unexpected indentation (expected 8, actual 12) (parameter-list-wrapping) // arguments
Unexpected indentation (expected 4, actual 8) (parameter-list-wrapping) // last line
```

## Steps to Reproduce
Try to run lint on the given example, in case of formatting - this gets reformatted wrongly, which then causes `indent` rule to fail

## Your Environment

* Version of ktlint used: 0.38.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): CLI

**Comment by mikeholler** [comment]
I'm also experiencing this.
